src/api/stats/sonic/getShadowPositionPrices.js

```diff
@@ -0,0 +1,16 @@
+const getUniV3PositionPrices = require('../common/getUniV3PositionPrices');
+const pools = require('../../../data/sonic/shadowLpPools.json');
+
+const getShadowPositionPrices = async tokenPrices => {
+  const params = {
+    pools: pools,
+    tokenPrices: tokenPrices,
+    chainId: 146,
+    beefyHelper: '0x70FcD79981f16277513030400a1f9fBc32A64C83',
```

Comment:
Is this address correct? Nothing is deployed there on Sonic. Can you make sure to verify too?